src/script/auth/component/AccountForm.tsx

```diff
@@ -230,12 +233,13 @@ const AccountForm = ({account, ...props}: Props & ConnectedProps & DispatchProps
         }}
         markInvalid={!validInputs.terms}
         name="accept"
+        id="accept"
         required
         checked={registrationData.termsAccepted}
         data-uie-name="do-terms"
         style={{justifyContent: 'center'}}
       >
-        <CheckboxLabel>
+        <CheckboxLabel htmlFor="name">
```

Comment:
Is it really for the `name` input? It feels weird, doesn't it?

Reply:
ach, you are right - it's for `accept` checkbox - I'll fix it